toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -136,7 +138,8 @@ intltool-0.51.0-7.cm2.noarch.rpm
 itstool-2.0.6-4.cm2.noarch.rpm
 kbd-2.2.0-1.cm2.x86_64.rpm
 kbd-debuginfo-2.2.0-1.cm2.x86_64.rpm
-kernel-headers-5.15.138.1-4.cm2.noarch.rpm
+kernel-cross-headers-5.15.138.1-5.cm2.noarch.rpm
+kernel-headers-5.15.138.1-5.cm2.noarch.rpm
```

Comment:
I think that the kernel auto-upgrade pipeline logic is somewhere in this Repo. We should update it to increment this new package.

Reply:
I think I found the file we would need to change; 

https://dev.azure.com/mariner-org/mariner/_search?action=contents&text=KernelCVE.json&type=code&lp=code-Project&filters=ProjectFilters%7Bmariner%7DRepositoryFilters%7BCBL-Mariner-Pipelines%7D&pageSize=25&result=DefaultCollection/mariner/CBL-Mariner-Pipelines/GBmaster//spec-auto-patch-update/config/KernelCVE.json

We would need to make the configuration aware of kernel-cross-headers-*. so that UpdateKernel.py catches it.

https://dev.azure.com/mariner-org/mariner/_search?action=contents&text=CblMarinerKernelRepo&type=code&lp=code-Project&filters=ProjectFilters%7Bmariner%7DRepositoryFilters%7BCBL-Mariner-Pipelines%7D&pageSize=25&result=DefaultCollection/mariner/CBL-Mariner-Pipelines/GBmaster//spec-auto-patch-update/tools/UpdateKernel.py

Reply:
Amazing, thank you Cameron! I'll look into this.

Reply:
Here's the [update to the auto-updater](https://dev.azure.com/mariner-org/mariner/_git/CBL-Mariner-Pipelines/pullrequest/17200).